### PR TITLE
cleanup signatures doc a bit

### PR DESCRIPTION
**Pull request by whyrusleeping**
Cleaning up the signatures doc a bit to make it clear whats needed for someone implementing Filecoin. 

Additionally, including a few random tweaks to other parts of the spec i noticed.

### CODE REVIEW

File: signatures.md [hunk 1]
```diff
@@ -123,93 +96,3 @@ sig SignatureBytes = [0x30][len][0x02][r][indicator][s][indicator][recovery]
...
-```
-
-[Current Filecoin Implementation](https://github.com/filecoin-project/go-filecoin/blob/ab60f73af1f86a954ef41e3252ef52a99066bbe2/types/signed_message.go#L25)
-
-
-### Github
-
-<https://github.com/filecoin-project/specs/issues/131>
-
-### External References
```
Comment (dignifiedquire) [inline signatures.md]: as mentioned in person, please keep the references to the specs and rfcs either here or in a general "Reference" section

File: sync.md [hunk 2]
```diff
@@ -164,6 +164,11 @@ func (syncer *Syncer) SyncCaughtUp(maybeHead TipSet) error {
 		return err
 	}
 
+      chain.ForEach(func(b Block) {
+        // FetchMessages should use bitswap to fetch any messages we don't have locally
+        FetchMessages(b)
```
Comment (dignifiedquire) [inline sync.md]: seems like an unrelated change

**Comment by whyrusleeping** [comment]
alright, left the reference to the eliptic curve cryptography spec.
